src/formio.common.ts

```diff
@@ -31,6 +31,36 @@ export interface FormioForm {
   submissionAccess?: AccessSetting[];
 }
 
+export interface GridColumn {
```

Comment:
@vlad-shusterman could you please move all grid related interfaces to separate files and give these files semantically correct name and place them to semantically correct folders? As I see `formio.common.ts` is more for some formio.js missing types and grid interfaces doesn't seem right in this file